---
layout: page
title: Projects
subtitle: Our research vision
desc: About me.
permalink: "/about/"
---
<div class="pretty-links">

<div class="lead lead-about">

Our research program combines **plant genomics** and **plant-microbe interactions**. Our vision is to combine new technologies, especially genomics with fundamental biology and translational research to make an impact on agriculture and food security. 

{::nomarkdown} <figure class="site-profile"> <img src="{{ site.baseurl }}/assets/img/profile.png"> </figure> {:/}

Driven by these principles,

·      we continue to develop wheat as model organism providing tools for the community

·      we apply genomic analyses across the plant kingdom to understand evolution of immunity

·      we work on understanding fundamental principles of plant immune receptor evolution and function 

Our long-term goal is to engineer plant immunity and plant immune receptors with new pathogen recognition specificities.

</div>

---

## **Mutational genomics to enhance disease resistance**

 

We have previously identified 34 mutant lines with enhanced adult plant resistance phenotypes against yellow rust. Our goal is to identify mutations linked to resistance and clone genes with causative mutations.

Initially, we identified 34 mutant lines in the US field trials. Now this collection has been re-grown for 3rd year phenotyping in the UK confirming broad spectrum resistance in 16 lines. The collection is also being evaluated in the fields in Kenya (yellow rust and stem rust) through collaboration with CIMMYT and the Gates foundation.

We deployed exome capture and bulk segregant analysis of mutations in F2 population to map genetic interval co-segregating with resistance in the first mutant line. Utilizing EI’s DRAGEN Bio-IT processor (Edico Genomics), we were able to reduce the processing time from days to minutes per sample. For several other mutant lines, we have advanced genetic material at TSL to the F2/3 generations and scored additional populations in the field in May 2017.

## **Evolution of plant immune receptors**

 

Plant NLR immune receptors enable plants to recognize diverse pathogens. Plant deploy multiple mechanisms of NLR diversification and regulation (Baggs, Dagdas and Krasileva, Curr Opinion 2017). Our evolutionary analyses of nine grass species (Setaria, Brachypodium, barley, rice, maize, wheat, and two wild wheat progenitors) demonstrated diverse fusions of genes originating from different chromosomes to a specific clade of plant NLR immune receptors (Bailey et al, in review). Based on current data, the most likely mechanism is non-homologous recombination where a nicking of DNA right after an NLR is repaired with DNA from another genomic location.  

In comparison to the Poaceae, other plant families have lower numbers of NLRs. Analysis of nearly 80 plant genomes revealed several independent events of lineage specific NLR contractions. NLR phylogenies across diverse families also revealed conservation of subclasses of NLRs. We propose that these conserved NLRs are candidates for a minimal functional plant immunity. 

Finally, we have identified maize and Sorghum lineages as a prime candidates for additional evolutionary analyses. Unlike other grasses, maize exhibits a putative reduction in NLR genes. We are applying targeted re-seuqncing with PacBio to determine if this reduction is due to domestication, inbreeding, or extensive balancing selection that maintains fewer loci in each genome.

---

## Our ongoing research aims

1. Understand the Effects of domestication, hybridization and inbreeding on NLR immune receptors.
2. Engineering plant immunity using CRISPR and synthetic biology.

</div>