---
layout: page
title: Projects
subtitle: Our research vision
desc: About me.
permalink: "/about/"
---
<div class="pretty-links">

<div class="lead lead-about">

Our research program combines **plant genomics** and **plant-microbe interactions**. Our vision is to combine new technologies, especially genomics with fundamental biology and translational research to make an impact on agriculture and food security.

</div>

---

## **Mutational genomics to enhance disease resistance**

We have previously identified 34 mutant wheat lines with enhanced adult plant resistance against a devastating wheat pathogen called [wheat yellow (stripe) rust](https://en.wikipedia.org/wiki/Wheat_yellow_rust). Our goal is to identify mutations linked to resistance and clone genes with causative mutations.

{::nomarkdown} <figure class="site-profile"> <img src="{{ site.baseurl }}/assets/img/profile.png"> </figure> {:/}

We identified our candidate mutant lines directly in the US field trials. We tested and confirmed resistance in a subset of lines across three continents, America (California), Europe (UK) and Africa (Kenya). In the UK, we have been testing resistance for four years confirming broad spectrum resistance against yellow rust in 16 lines. All 34 lines were evaluated in the fields in Kenya (for resistance against stem rust) through collaboration with [Durable Resistance in Wheat Gates project](http://www.wheatrust.cornell.edu/).

We deployed exome capture and bulk segregant analysis of mutations in F2 population to map genetic interval co-segregating with resistance in the first mutant line. Wheat has a complex genome, 5x larger than human genome, and procesing wheat genomic data remains a computational challenge. Utilizing DRAGEN Bio-IT processor (Edico Genomics) availabe at [Earlham Institute](http://www.earlham.ac.uk/dragen-processor), we were able to reduce the processing time from days to minutes per sample.

We have advanced genetic material to the F2/3 generations and are scoring them in the field, aiming to de-cipher the resistance genes within the next year.

The next step after we know what mutations have beneficial effect on disease resistance is introducing them to elite varieties grown across the world. We are taking two complimentary approaches 1) mining natural variation in wheat for alleles similar to our induced changes from projects such as [WHEALBI ]()where the data is available on >500 wheat lines representing worldwide and historical diversity 2) introducing precise changes into elite varieties using genome editing.

And of course, we are planning more mutational genomics screens, both in the field and in the lab.

## **Evolution and function of plant immune receptors**

Plant NLR immune receptors enable plants to recognize diverse pathogens. Plant deploy multiple mechanisms of NLR diversification and regulation. [Here is our review on this topic]().

We are interested to understand evolution of NLRs on multiple levels: 1) within single organism. How much diversity is generated in each generation? 2) On a population levels. How much immune receptor diversity does a field of grass have? 3) Across species 4) Across kingdom We are keen to examine effects of domestication and hybridization on NLR evolution on all these levels.

[Our recent evolutionary analyses]() of NLRs in nine grass species (Setaria, Brachypodium, barley, rice, maize, wheat, and two wild wheat progenitors) demonstrated that a specific class of NLRs is prone to diverse fusions with other genes, often originating from different chromosomes. Such fusions are utilized by NLRs as baits for the pathogen-derived molecules. Based on current data, we think that the most likely mechanism ifor this s non-homologous recombination where a nicking of DNA right after an NLR is repaired with DNA from another genomic location. We are working to uncover the exact mechanism of how NLRs can tolerate new gene fusions, so we can hack this evolutionary mechanism for designing new immune receptor.

{::nomarkdown} <figure class="site-profile"> <img src="{{ site.baseurl }}/assets/img/profile.png"> </figure> {:/}

In comparison to the Poaceae, other plant families have lower numbers of NLRs. Analysis of nearly 80 plant genomes revealed several independent events of lineage specific NLR contractions. NLR phylogenies across diverse families also revealed conservation of subclasses of NLRs. We propose that these conserved NLRs are candidates for a minimal functional plant immunity.

We have identified maize and Sorghum lineages as a prime candidates for additional evolutionary analyses. Unlike other grasses, maize exhibits a putative reduction in NLR genes. We are applying targeted re-seuqncing with PacBio to determine if this reduction is due to domestication, inbreeding, or extensive balancing selection that maintains fewer loci in each genome.

---

## Our ongoing research aims

1. Understand the effects of domestication, hybridization and inbreeding on NLR immune receptors.
2. Engineering plant immunity using CRISPR and synthetic biology.

</div>