---
layout: page
title: Ksenia V Krasileva
subtitle: Ksenia V Krasileva
desc: Group leader
permalink: "/people/kvk/"
---
<h3>Ksenia V Krasileva</h3><p>Group Leader (from July 2018 Assistant Professor at University of California Berkeley). A scientist interested in Genomics and Plant Immunity. </p><p>ORCHID:                     	0000-0002-1679-0700</p><p>Twitter: 		@kseniakrasileva</p><p>Github (code):		https://github.com/krasileva-group </p><p>E-mail: 			Ksenia.Krasileva@earlham.ac.uk</p><p><br>                       </p><p>2011	PhD	Microbiology, Designated Emphasis in Genomics and Computational Biology University of California, Berkeley, USA</p><p>2005	BS	Plant Genetics and Microbiology	University of California, Berkeley, USA</p><br><p>PREVIOUS POSITIONS</p><p>2012 - 2014: USDA AFRI-NIFA Postdoctoral Fellow, Department of Plant Science, University of California, Davis, USA w/Jorge Dubcovsky</p><p>2011 - 2012: Howard Hughes Medical Institute Postdoctoral Scholar, Department of Plant Science, University of California, Davis, USA. w/Jorge Dubcovsky</p><p>2005 - 2011: Graduate Student Researcher, Department of Plant and Microbial Biology, University of California Berkeley w/Brian Staskawicz</p><p>2003 - 2005: Undergraduate Student Researcher Department of Plant and Microbial Biology, University of California Berkeley w/Steven Lindow</p><p>MAJOR GRANTS</p><p>2017-2022 ERC Starter Grant "Mechanisms of Immune Receptor Diversification in Cereals" Sole PI €1,500,000</p><p>2016-2019 BBSRC (BB/N016106/1) "Rapid identification of genes and pathways that increase resistance to yellow rust disease of wheat" Sole PI £590,000</p><p>2012-2017 BBSRC (BB/J003743/1) LOLA Longer and Larger Grants "Triticeae Genomics for Sustainable Agriculture" co-I £3,400,000</p><p>2014-2019 European Commission FP7 Collaborative Project (12 groups) "Wheat and Barley Legacy for Breeding Improvement" co-I £390,893</p><p>2016-2019 Bilateral BBSRC-Embrapa: Exploiting natural and induced variation to increase Fusarium head blight and brusone resistance in wheat co-I £25,296</p><p>2012-2014 AFRI-NIFA Postdoctoral Fellowship "Functional Genomic Tools for Wheat" $130,000</p><p>MEMBERSHIPS OF SCIENTIFIC SOCIETIES</p><p>2016 - current: International Society - Molecular Plant Microbe Interactions</p><p>2015 - 2018: SUSTAIN COST Action "Pathogen-informed strategies for sustainable broad-spectrum crop resistance"</p><p>2014 - 2018: Research Network "Norwich Rust Group"</p><p>TEACHING</p><p>2017. Lecturer "Summer school in plant-microbe interactions", The Sainsbury Laboratory, Norwich UK</p><p>2008-2010. Guest Lecturer "Agriculture and Society", University of California, Berkeley</p><p>2007. Teaching Assistant "Introduction to Programming for Bioinformatics", University of California, Berkeley</p><p>2006-2007. Graduate Student Instructor, University of California Berkeley</p><p>STUDENT MENTORSHIP</p><p>2016-current Erin Baggs (graduate student)</p><p>2016-2017 Marcus Watkins (MSc student, completed)</p><p>2016-2017 Amelie Heckmann (pre-doc, now PhD with Fiona Doohan, University College Dublin)</p><p>2013-2014 Hans Vasquez-Gross (pre-doc, now PhD with Jorge Dubcovsky, UC Davis)</p><p>2012-2013 Vince Buffalo (pre-doc, now PhD with Jeff Ross-Ibarra, UC Davis)</p><p>2010 Kelsie Moryoka (undergradute, now RA with Ben Blackman, UC Berkeley)</p><p>2009-2010 Connie Cheng (undergraduate)</p><p>2008-2009 Margie Payumo (undergraduate)</p><p>PATENTS</p><p>1.	December 2016 Schudoma C, Clark M and Krasileva KV British Patent Application No. 1620789.6 Gluten-seq Earlham Institute </p><p>PUBLICATIONS (23 total, 6 first/co-first, 4 corresponding author)</p><p> http://scholar.google.co.uk/citations?user=ATjjt-sAAAAJ&amp;hl=en</p><p>2018</p><p>1.	Bailey PC, Schudoma C, Jackson W, Baggs E, Dagdas G, Haerty W, Moscou M, Krasileva KV "Dominant integration locus drives continuous diversification of plant immune receptors with exogenous domain fusions" Genome Biology 2018 19:23 (first at BioRxiv) https://doi.org/10.1186/s13059-018-1392-6 - 3 citations</p><p>2017</p><p>2.	Krasileva KV "Secrets of a hardy crop: New genomic resources for pearl millet are revealing how crops stand up to challenging environments" Nature Biotechnology, 35(10) Oct 2017 News and Views https://doi.org/10.1038/nbt.3984</p><p>3.	Baggs E, Dagdas G, Krasileva KV "NLR diversity, helpers and integrated domains: making sense of the NLR IDentity" Curr Opinion in Plant Biology, 2017 38:59-67. Review https://doi.org/10.1016/j.pbi.2017.04.012 - 5 citations</p><p>4.	Bevan MW, Uauy C, Wulff BBH, Zhou1 J, Krasileva KV, and Clark M "Genomic Innovation for Crop Improvement" Nature, Mar 15;543(7645):346-354. Review</p><p>https://doi.org/10.1038/nature22011  - 29 citations</p><p>5.	Krasileva KV, Vasquez-Gross H, Bailey P, Paraiso F, Clissold L, Howell T, Ramirez-Gonzalez R, Wang X, Ayling S, Fosker C, Phillips A, Uauy C, Dubcovsky "Uncovering hidden variation in polyploid wheat." Proceeding of National Academy of Sciences U S A, 2017 Feb 7;114(6):E913-E921 https://doi.org/10.1073/pnas.1619268114  - 46 citations</p><p>6.	Clavijo BJ, Venturini L, Schudoma C, Accinelli GG, Kaithakottil G, Wright J, Borrill P, Kettleborough G, Heavens D, Chapman H, Lipscombe J, Barker T, Lu FH, McKenzie N, Raats D, Ramirez-Gonzalez RH, Coince A, Peel N, Percival-Alwyn L, Duncan O, Trösch J, Yu G, Bolser DM, Namaati G, Kerhornou A, Spannagl M, Gundlach H, Haberer G, Davey RP, Fosker C, Palma FD, Phillips AL, Millar AH, Kersey PJ, Uauy C, Krasileva KV, Swarbreck D, Bevan MW, Clark MD "An improved assembly and annotation of the allohexaploid wheat genome identifies complete families of agronomic genes and provides genomic evidence for chromosomal translocations." Genome Res. 2017 May;27(5):885-896 (first at BioRxiv) https://doi.org/10.1101/gr.217117.116 - 57 citations</p><p>2016</p><p>7.	Sarris PF, Cevik V, Dagdas G, Jones JD, Krasileva KV "Comparative analysis of plant immune receptor architectures uncovers host proteins likely targeted by pathogens" BMC Biology 2016 14(1):8. - 50 citations https://doi.org/10.1186/s12915-016-0228-7</p><p>8.	Petre B, Saunders DG, Sklenar J, Lorrain C, Krasileva KV, Win J, Duplessis S, Kamoun S. "Heterologous Expression Screens in Nicotiana benthamiana Identify a Candidate Effector of the Wheat Yellow Rust Pathogen that Associates with Processing Bodies" PLoS One 2016 11(2):e0149035.  (first at BioRxiv) https://doi.org/10.1371/journal.pone.0149035 - 23 citations</p><p>2015</p><p>9.	Schwessinger B, Bart R, Krasileva KV and Coaker G. "Focus issue on plant immunity: from model systems to crop species." Front Plant Sci. 2015 6:195. https://doi.org/10.3389/fpls.2015.00195  - 5 citations</p><p>10.	Wu CH, Krasileva KV, Banfield MJ, Terauchi R, Kamoun S.  "The "sensor domains" of plant NLR proteins: more than decoys?" Front Plant Sci. 2015 6:134. https://doi.org/10.3389/fpls.2015.00134 - 29 citations</p><p>2014</p><p>11.	Henry IM, Nagalakshmi U, Lieberman MC, Ngo KJ, Krasileva KV, Vasquez-Gross H, Alina Akunova A, Akhunov E, Dubcovsky J, Tai TH and Comai L "Efficient genome-wide detection and cataloguing of EMS-induced mutations using exome capture and next-generation sequencing" Plant Cell 2014 26(4):1382-1397. https://doi.org/10.1105/tpc.113.121590 - 91 citations</p><p>2013</p><p>12.	Krasileva KV, Buffalo V, Bailey P, Pearce S, Ayling S, Tabbita F, Soria M, Wang S, Consortium I, Akhunov E, Uauy C, Dubcovsky J "Separating homeologs by phasing in the tetraploid wheat transcriptome". Genome Biol. 2013 14(6):R66. https://doi.org/10.1186/gb-2013-14-6-r66 - 83 citations</p><p>2012</p><p>13.	Steinbrenner AD, Goritschnig S, Krasileva KV, Schreiber KJ, Staskawicz BJ. "Effector recognition and activation of the Arabidopsis thaliana NLR innate immune receptors." Cold Spring Harb Symp Quant Biol. 2012 77:249-57.</p><p>14.	Bart R, Cohn M, Kassen A, McCallum EJ, Shybut M, Petriello A, Krasileva K, Dahlbeck D, et al. "High-throughput genomic sequencing of cassava bacterial blight strains identifies conserved effectors to target for durable resistance." PNAS, 2012 109(28):E1972-9.</p><p>15.	Goritschnig S, Krasileva KV, Dahlbeck D, Staskawicz BJ. "Computational prediction and molecular characterization of an oomycete effector and the cognate Arabidopsis resistance gene" PLoS Genetics 2012 8(2):e1002502. OPEN ACCESS</p><p>16.	Win J, Krasileva KV, Kamoun S, Shirasu K, Staskawicz BJ, Banfield MJ. (2012) "Sequence divergent RXLR effectors share a structural fold conserved across plant pathogenic oomycete species" PLoS Pathogens 8(1): e1002400. OPEN ACCESS</p><p>2011</p><p>17.	Krasileva KV, Zheng C, Leonelli L, Goritschnig S, Dahlbeck D, Staskawicz BJ. (2011) "Global analysis of Arabidopsis / downy mildew interactions reveals prevalence of incomplete resistance and rapid evolution of pathogen recognition" PLoS One, 6: e28765. OPEN ACCESS</p><p>18.	Zhao B, Dahlbeck D, Krasileva KV, Fong RW, Staskawicz BJ. (2011) "Computational and Biochemical Analysis of the Xanthomonas Effector AvrBs2 and Its Role in the Modulation of Xanthomonas Type Three Effector Delivery." PLoS Pathogens. 12 :e1002408. OPEN ACCESS</p><p>19.	Chou S*, Krasileva KV*, Holton J, Steinbrenner A, Alber T, Staskawicz BJ. (2011) "Hyaloperonospora arabidopsidis ATR1 effector is a repeat protein with distributed recognition surfaces" PNAS 108(32): 13323-8. *equal first author contribution</p><p>20.	Potnis N*, Krasileva K*, Chow V, Almeida NF Jr, Patil PB, et al. (2011) "Comparative Genomics Reveals Diversity among Xanthomonads Infecting Tomato and Pepper". BMC Genomics, 12(1): 146.*equal first author contribution OPEN ACCESS</p><p>2010</p><p>21.	Krasileva KV, Dahlbeck D, Staskawicz BJ. (2010) "Activation of an Arabidopsis Resistance Protein Is Specified by the in Planta Association of Its Leucine-Rich Repeat Domain with Cognate Oomycete Effector". Plant Cell, 22:1-16.</p><p>22.	Dulla GF, Krasileva KV, Lindow SE. (2010) "Interference of Quorum Sensing in Pseudomonas syringae by Bacterial Epiphytes that Limit Iron Availability". Environmental Microbiology, 12(6): 1762-74.</p><p>2007</p><p>23.	Win J, Morgan W, Bos J, Krasileva KV, Cano LM, Chaparro-Garcia A, Ammar L, Staskawicz BJ, Kamoun S. (2007) "Adaptive Evolution Has Targeted the C-terminal Domain of the RXLR Effectors of Plant Pathogenic Oomycetes". Plant Cell, 19: 2349-69</p><div class="projects"><div class="grid no-gutters"><div class="unit one-third"><div class="project">
      </div>
    </div>
  </div><!-- grid -->
</div>