---
layout: page
title: Ksenia V Krasileva
subtitle: Ksenia V Krasileva
desc: Group leader
permalink: "/people/kvk/"
---
<h3>Group Leader at Earlham Institute and Sainsbury Laboratory (from July 2018 Assistant Professor at University of California Berkeley). </h3><h3>A scientist interested in Genomics and Plant Immunity. <br></h3><p><a href="https://scholar.google.co.uk/citations?user=ATjjt-sAAAAJ&amp;hl=en">Google Scholar profile</a>&nbsp; ---&nbsp; <a href="https://docs.google.com/document/d/1IJ2iT1vob-NtTVmVExoC3_5b4AA3CxEm488UheecRVc/edit?usp=sharing">Download full CV</a><br></p><p><a href="https://twitter.com/kseniakrasileva">@kseniakrasileva</a>&nbsp;&nbsp; Ksenia.Krasileva@earlham.ac.uk</p><p><b>2011	PhD	</b>Microbiology, Designated Emphasis in Genomics and Computational Biology University of California, Berkeley, USA</p><p><b>2005	BS	</b>Plant Genetics and Microbiology	University of California, Berkeley, USA</p><br><p><b>STUDENT MENTORSHIP</b></p><p>2016-current Erin Baggs (graduate student)</p><p>2016-2017 Marcus Watkins (MSc student, completed)</p><p>2016-2017 Amelie Heckmann (pre-doc, now PhD with Fiona Doohan, University College Dublin)</p><p>2013-2014 Hans Vasquez-Gross (pre-doc, now PhD with Jorge Dubcovsky, UC Davis)</p><p>2012-2013 Vince Buffalo (pre-doc, now PhD with Jeff Ross-Ibarra, UC Davis)</p><p>2010 Kelsie Moryoka (undergradute, now RA with Ben Blackman, UC Berkeley)</p><p>2009-2010 Connie Cheng (undergraduate)</p><p>2008-2009 Margie Payumo (undergraduate)</p><p><b>PUBLICATIONS (23 total, 6 first/co-first, 4 corresponding author)</b></p><p><u>Krasileva lab member names underlined</u><b><br></b></p><p>2018</p><p>1.	<u>Bailey PC</u>, <u>Schudoma C</u>, Jackson W, <u>Baggs E</u>, <u>Dagdas G</u>, Haerty W, Moscou M, <b>Krasileva KV</b> "<a href="https://doi.org/10.1186/s13059-018-1392-6">Dominant integration locus drives continuous diversification of plant immune receptors with exogenous domain fusions</a>" Genome Biology 2018 19:23 (first at BioRxiv)<br></p><p>2017</p><p>2.	<b>Krasileva KV</b> "<a href="https://doi.org/10.1038/nbt.3984">Secrets of a hardy crop: New genomic resources for pearl millet are revealing how crops stand up to challenging environments</a>" Nature Biotechnology, 35(10) Oct 2017 News and Views <br></p><p>3.	<u>Baggs E</u>, <u>Dagdas G</u>, <b>Krasileva KV</b> "<a href="https://doi.org/10.1016/j.pbi.2017.04.012">NLR diversity, helpers and integrated domains: making sense of the NLR IDentity" Curr Opinion in Plant Biology</a>", 2017 38:59-67. Review <br></p><p>4.	Bevan MW, Uauy C, Wulff BBH, Zhou1 J, Krasileva KV, and Clark M "<a href="https://doi.org/10.1038/nature22011">Genomic Innovation for Crop Improvement</a>" Nature, Mar 15;543(7645):346-354. Review</p><p>5.	Krasileva KV, Vasquez-Gross H, Bailey P, Paraiso F, Clissold L, Howell T, Ramirez-Gonzalez R, Wang X, Ayling S, Fosker C, Phillips A, Uauy C, Dubcovsky "<a href="https://doi.org/10.1073/pnas.1619268114">Uncovering hidden variation in polyploid wheat.</a>" Proceeding of National Academy of Sciences U S A, 2017 Feb 7;114(6):E913-E921<br></p><p>6.	Clavijo BJ, Venturini L, <u>Schudoma C</u>, Accinelli GG, Kaithakottil G, Wright J, Borrill P, Kettleborough G, Heavens D, Chapman H, Lipscombe J, Barker T, Lu FH, McKenzie N, <u>Raats D</u>, Ramirez-Gonzalez RH, Coince A, Peel N, Percival-Alwyn L, Duncan O, Tr√∂sch J, Yu G, Bolser DM, Namaati G, Kerhornou A, Spannagl M, Gundlach H, Haberer G, Davey RP, Fosker C, Palma FD, Phillips AL, Millar AH, Kersey PJ, Uauy C, <b>Krasileva KV</b>, Swarbreck D, Bevan MW, Clark MD "<a href="https://doi.org/10.1101/gr.217117.116">An improved assembly and annotation of the allohexaploid wheat genome identifies complete families of agronomic genes and provides genomic evidence for chromosomal translocations</a>." Genome Res. 2017 May;27(5):885-896 (first on BioRxiv)&nbsp; <br></p><p>2016</p><p>7.	Sarris PF, Cevik V, <u>Dagdas G</u>, Jones JD, <b>Krasileva KV</b> "<a href="https://doi.org/10.1186/s12915-016-0228-7">Comparative analysis of plant immune receptor architectures uncovers host proteins likely targeted by pathogens</a>" BMC Biology 2016 14(1):8.<br></p><p>8.	Petre B, Saunders DG, Sklenar J, Lorrain C, Krasileva KV, Win J, Duplessis S, Kamoun S. "<a href="https://doi.org/10.1371/journal.pone.0149035">Heterologous Expression Screens in Nicotiana benthamiana Identify a Candidate Effector of the Wheat Yellow Rust Pathogen that Associates with Processing Bodies</a>" PLoS One 2016 11(2):e0149035.  (first on BioRxiv)&nbsp; <br></p><p>2015</p><p>9.	Schwessinger B, Bart R, <b>Krasileva KV </b>and Coaker G. "<a href="https://doi.org/10.3389/fpls.2015.00195">Focus issue on plant immunity: from model systems to crop species</a>." Front Plant Sci. 2015 6:195. <br></p><p>10.	Wu CH, <b>Krasileva KV</b>, Banfield MJ, Terauchi R, Kamoun S.  "<a href="https://doi.org/10.3389/fpls.2015.00134">The "sensor domains" of plant NLR proteins: more than decoys?</a>" Front Plant Sci. 2015 6:134.&nbsp; <br></p><p>2014</p><p>11.	Henry IM, Nagalakshmi U, Lieberman MC, Ngo KJ, <b>Krasileva KV</b>, Vasquez-Gross H, Alina Akunova A, Akhunov E, Dubcovsky J, Tai TH and Comai L "<a href="https://doi.org/10.1105/tpc.113.121590">Efficient genome-wide detection and cataloguing of EMS-induced mutations using exome capture and next-generation sequencing</a>" Plant Cell 2014 26(4):1382-1397. <br></p><p>2013</p><p>12.	<b>Krasileva KV</b>, Buffalo V, Bailey P, Pearce S, Ayling S, Tabbita F, Soria M, Wang S, Consortium I, Akhunov E, Uauy C, Dubcovsky J "<a href="https://doi.org/10.1186/gb-2013-14-6-r66">Separating homeologs by phasing in the tetraploid wheat transcriptome</a>". Genome Biol. 2013 14(6):R66. <br></p><p>2012</p><p>13.	Steinbrenner AD, Goritschnig S, <b>Krasileva KV</b>, Schreiber KJ, Staskawicz BJ. "Effector recognition and activation of the Arabidopsis thaliana NLR innate immune receptors." Cold Spring Harb Symp Quant Biol. 2012 77:249-57.</p><p>14.	Bart R, Cohn M, Kassen A, McCallum EJ, Shybut M, Petriello A, <b>Krasileva K</b>, Dahlbeck D, et al. "High-throughput genomic sequencing of cassava bacterial blight strains identifies conserved effectors to target for durable resistance." PNAS, 2012 109(28):E1972-9.</p><p>15.	Goritschnig S, <b>Krasileva KV</b>, Dahlbeck D, Staskawicz BJ. "Computational prediction and molecular characterization of an oomycete effector and the cognate Arabidopsis resistance gene" PLoS Genetics 2012 8(2):e1002502. <br></p><p>16.	Win J, <b>Krasileva KV</b>, Kamoun S, Shirasu K, Staskawicz BJ, Banfield MJ. (2012) "Sequence divergent RXLR effectors share a structural fold conserved across plant pathogenic oomycete species" PLoS Pathogens 8(1): e1002400. <br></p><p>2011</p><p>17.	<b>Krasileva KV</b>, Zheng C, Leonelli L, Goritschnig S, Dahlbeck D, Staskawicz BJ. (2011) "Global analysis of Arabidopsis / downy mildew interactions reveals prevalence of incomplete resistance and rapid evolution of pathogen recognition" PLoS One, 6: e28765.<br></p><p>18.	Zhao B, Dahlbeck D, <b>Krasileva KV</b>, Fong RW, Staskawicz BJ. (2011) "Computational and Biochemical Analysis of the Xanthomonas Effector AvrBs2 and Its Role in the Modulation of Xanthomonas Type Three Effector Delivery." PLoS Pathogens. 12 :e1002408. <br></p><p>19.	Chou S*, <b>Krasileva KV</b>*, Holton J, Steinbrenner A, Alber T, Staskawicz BJ. (2011) "Hyaloperonospora arabidopsidis ATR1 effector is a repeat protein with distributed recognition surfaces" PNAS 108(32): 13323-8. *equal first author contribution</p><p>20.	Potnis N*, <b>Krasileva K</b>*, Chow V, Almeida NF Jr, Patil PB, et al. (2011) "Comparative Genomics Reveals Diversity among Xanthomonads Infecting Tomato and Pepper". BMC Genomics, 12(1): 146.*equal first author contribution <br></p><p>2010</p><p>21.	<b>Krasileva KV</b>, Dahlbeck D, Staskawicz BJ. (2010) "Activation of an Arabidopsis Resistance Protein Is Specified by the in Planta Association of Its Leucine-Rich Repeat Domain with Cognate Oomycete Effector". Plant Cell, 22:1-16.</p><p>22.	Dulla GF, <b>Krasileva KV,</b> Lindow SE. (2010) "Interference of Quorum Sensing in Pseudomonas syringae by Bacterial Epiphytes that Limit Iron Availability". Environmental Microbiology, 12(6): 1762-74.</p><p>2007</p><p>23.	Win J, Morgan W, Bos J, <b>Krasileva KV</b>, Cano LM, Chaparro-Garcia A, Ammar L, Staskawicz BJ, Kamoun S. (2007) "Adaptive Evolution Has Targeted the C-terminal Domain of the RXLR Effectors of Plant Pathogenic Oomycetes". Plant Cell, 19: 2349-69</p><div class="projects"><div class="grid no-gutters"><div class="unit one-third"><div class="project">
      </div>
    </div>
  </div><!-- grid -->
</div>