---
layout: page
title: Toolbox
subtitle: KVK Toolbox
desc: Toolbox
permalink: "/toolbox/"
---
<h2>Code&nbsp;</h2><p>Our code is maintained at <a href="https://github.com/krasileva-group/">GitHub</a>. https://github.com/krasileva-group/</p><p>Available under MIT license.</p><h3><a href="https://github.com/krasileva-group/plant_rgenes">Plant R-genes</a></h3><p>https://github.com/krasileva-group/plant_rgenes</p><p>Scripts to annotate&nbsp;<span style="font-size: 1rem;">Pfam domains and extract NLR plant immune receptors and their architectures as published in <a href="https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-016-0228-7">Sarris et al BMC Biology 2016</a>.</span></p><h3><a href="https://github.com/krasileva-group/scvep">SCVEP</a></h3><p>https://github.com/krasileva-group/scvep</p><p>Annotation-free variant effect prediction. It is based on blasting DNA sequence surrounding the SNP against protein database and resulting alignment is used to predict likely effect of the SNP on the coding sequence.<br></p><h3><a href="https://github.com/krasileva-group/wheat_tilling_pub">Wheat TILLING PNAS 2017</a></h3><p>https://github.com/krasileva-group/wheat_tilling_pub</p><p>Collaborative set of scripts used in Krasileva et al 2017 Wheat TILLING publication. <a href="https://github.com/DubcovskyLab/wheat_tilling_pub" style="">Forked from repository</a>.</p><h3><a href="https://github.com/krasileva-group/tandem">Tandem</a></h3><p>https://github.com/krasileva-group/tandem</p><p>Identification of neighboring genes, used in Bailey et al. Genome Biology 2018.</p><h3><a href="https://github.com/krasileva-group/tgacv1-circos">TGACv1-circos</a></h3><p>https://github.com/krasileva-group/tgac-map</p><p>https://github.com/krasileva-group/tgacv1-circos</p><p>Scripts used to <a href="https://github.com/krasileva-group/tgac-map">order genome scaffolds</a> and to <a href="https://github.com/krasileva-group/tgacv1-circos">plot genetic map</a> of wheat used in Clavijo et al. 2017. <br></p><h3><a href="https://github.com/krasileva-group/lrr_crawler">LRR crawler</a></h3><p>https://github.com/krasileva-group/lrr_crawler</p><p>Extracting leucine Rich Repeat Motifs with regular expression. Currently, under development.<br></p><h2>Teaching Materials</h2><h3><a href="https://docs.google.com/document/d 19dTUt5jZ9LrXllPnIIkOzRDD9kaG2xGXWymH7ruiOTg/edit?usp=sharing">Phylogenetics tutorial with NLR-ID examples </a></h3><p><a href="https://docs.google.com/document/d%2019dTUt5jZ9LrXllPnIIkOzRDD9kaG2xGXWymH7ruiOTg/edit?usp=sharing">Link on Google Docs</a><br></p><p>Krasileva Group (Erin Baggs, Elisha Thynne and Ksenia Krasileva) assembled the tutorial and taught this module during TSL summer school 2017.<br></p>